platform-core-module/src/main/java/com/softicar/platform/core/module/program/AGProgramTable.java

```diff
@@ -33,6 +37,22 @@ public void customizeAttributeProperties(IEmfAttributeList<AGProgram> attributes
 			.setTitle(CoreI18n.PROGRAM)
 			.setImmutable(true)
 			.setPredicateMandatory(EmfPredicates.always());
+
+		attributes//
+			.addTransientAttribute(AGProgram.QUEUED_AT)
+			.setDisplayFactory(EmfDayTimeDisplay::new);
```

Comment:
Do we really need to explicitly define display factories here? Because this _should_ work out-of-the-box.
I am not certain though.

(same for other transient attributes below)

Reply:
Yes, it it necessary to define display factories.

Otherwise whole records are displayed:
![Screenshot_2022-01-18_14-49-04](https://user-images.githubusercontent.com/74466399/150527105-6b4fe2f3-37e3-41b1-b736-7351cfdb2dc9.png)
And note that column "Abort Requested" shows "false" instead of "No".

Reply:
Oh... so, our framework is less awesome than expected. Never mind.